### PR TITLE
data:kaushalnitb

### PR DESCRIPTION
**Pull request by kaushalnitb**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4492"></a>

### CODE REVIEW

**Review by priyankarpal — CHANGES_REQUESTED** [review]
hey, @kaushalnitb you added the wrong JSON code. Please check and fix them. You can check the full profile example code https://linkfree.eddiehub.io/docs/full-profile-example

File: data/kaushalnitb.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+           {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/kaushalnitb",
+      "icon": "FaTwitter"
+   },
+{
+      "name": "Follow me on github",
+      "url": "https://github.com/kaushalnitb",
+      "icon": "FaGitHub"
+   },
```
Comment (kumarsonsoff3) [inline data/kaushalnitb.json:22]: There is a trailing comma `,` which should be removed. 
```suggestion
   }
```

File: data/kaushalnitb.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Kaushal Kishor",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a DevOps engineer"
```
Comment (kumarsonsoff3) [inline data/kaushalnitb.json:5]: There is a missing comma `,` which should be added in the end. 
```suggestion
  "bio": "I am a DevOps engineer",
```

File: data/kaushalnitb.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "links": [
+  {
+  "name": "Sara Jaoude",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado",
+  "links": [
```
Comment (kumarsonsoff3) [inline data/kaushalnitb.json:12]: Your file contains duplicate keys, please remove these duplicates as suggested. 
```suggestion
```

**Pro Tip:** To make the requested changes directly, you can use the <kbd>Commit suggestion</kbd> button above.

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/kaushalnitb.json (hunk 1 above)
Comment (kaushalnitb) [reply data/kaushalnitb.json:22]: Thanks

File: data/kaushalnitb.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Kaushal Kishor",
```
Comment (priyankarpal) [inline data/kaushalnitb.json:2]: hey, @kaushalnitb you mistakenly added an extra code. kindly delete these from lines 1 to 6. if you want to add clear `json` code from scratch you can check the [official doc](https://linkfree.eddiehub.io/docs/full-profile-example) 

```diff
{
  "name": "Kaushal Kishor",
  "displayStatsPublic": true,
  "type": "personal",
  "bio": "I am a DevOps engineer",
  "links": [
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/kaushalnitb.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
```
Comment (priyankarpal) [inline data/kaushalnitb.json:1]: Just add `{` before the name.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/kaushalnitb.json [hunk 6]
```diff
@@ -0,0 +1,18 @@
+  {
+  "name": "Kaushal Kishor",
+  "displayStatsPublic": true,
```
Comment (priyankarpal) [inline data/kaushalnitb.json:3]: After a recent update, this code is no longer needed. You can delete this line `"displayStatsPublic": true,`

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Review by priyankarpal — APPROVED** [review]
looking good to me  🎉

File: data/kaushalnitb.json (hunk 6 above)
Comment (kaushalnitb) [reply data/kaushalnitb.json]: I Have updated the code.Kindly verify it now.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good! 👍 
Great collaboration everyone! 🎉

**Pull request merged**